hack/images/titus-sshd/run-titus-sshd

```diff
@@ -0,0 +1,41 @@
+#!/titus/sshd/bin/busybox sh
+
+write_etc_profile() {
+cat > /etc/profile <<'EOF'
+if [ -d /etc/profile.d ]; then
+  for i in /etc/profile.d/*.sh; do
+    if [ -r $i ]; then
+      . $i
+    fi
+  done
+  unset i
+fi
+EOF
+}
+
+setup_files() {
+  # These are the bare minimum set of files that
+  # must exist for ssh to work
+  [ -f /etc/passwd ] || touch /etc/passwd
```

Comment:
We should probably log if we had to do this.

Reply:
great idea